pipeline/outputs/elasticsearch.md

```diff
@@ -13,6 +13,7 @@ The **es** output plugin, allows to ingest your records into an [Elasticsearch](
 | Host | IP address or hostname of the target Elasticsearch instance | 127.0.0.1 |
 | Port | TCP port of the target Elasticsearch instance | 9200 |
 | Path | Elasticsearch accepts new data on HTTP query path "/\_bulk". But it is also possible to serve Elasticsearch behind a reverse proxy on a subpath. This option defines such path on the fluent-bit side. It simply adds a path prefix in the indexing HTTP POST URI. | Empty string |
+| compress | Set payload compression mechanism. Option available is 'gzip' | |
```

Comment:
This setting is identical (in functionality, configuration, etc) to the one for the http output. 

Therefore I ensured that the documentation was also identical. For the HTTP output see:
https://github.com/fluent/fluent-bit-docs/blob/master/pipeline/outputs/http.md?plain=1#L21

If you still would like a change:
- Could you sent as a suggestion? (see [point 6 here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) )
- Should I create a second PR so that the HTTP configuration becomes identical?

Note that the setting is called `compress` with lowercase, and by default there is no value. The only allowed option is `gzip`.

Reply:
@lecaros any chance to look at this again? 😊